### PR TITLE
Drop support for Python 3.6

### PR DESCRIPTION
**Pull request by arcondello**
I think that's all of the places, we dropped it from the README.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -46,7 +46,7 @@ install_requires =
     penaltymodel==0.16.6
     pyqubo==1.2.0
 packages = dwaveoceansdk
-python_requires = >=3.6
+python_requires = >=3.7
```
Comment (randomir) [inline setup.cfg:49]: You missed it in `classifiers`.

Comment (arcondello) [reply setup.cfg:49]: :flushed:

**Comment by JoelPasvolsky** [comment]
https://github.com/dwavesystems/dwave-ocean-sdk/blob/master/docs/overview/install.rst

**Review by randomir — APPROVED** [review]

**Comment by arcondello** [comment]
> https://github.com/dwavesystems/dwave-ocean-sdk/blob/master/docs/overview/install.rst

Ok, I am going to remove the specific version reference from that file.

**Comment by arcondello** [comment]
> https://github.com/dwavesystems/dwave-ocean-sdk/blob/master/docs/overview/install.rst: "Python versions 3.6 and higher are supported."

Handled by https://github.com/dwavesystems/dwave-ocean-sdk/pull/186

**Pull request merged**